src/constants/dicom/Tags.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * DICOM tags as constants
+ */
+module.exports = Object.freeze({
+    SEQUENCE_DELIMITATION_TAG: 0xfffee0dd,
+    ITEM_TAG: 0xfffee000,
+    ITEM_DELIMITATION_TAG: 0xfffee00d
```

Comment:
These are already in dictionary.js so I don't think we should duplicate them here.

Reply:
The dictionary would not work for this specific use case, because the idea was to have the name as constant in the code instead of the hex value or in case of the dictionary the tag as "(FFFE,E000)" String, for instance. I moved the 3 items into the class as constants, since we are not using them somewhere else. I hope that is ok for you.

Reply:
I'm not understanding why the dictionary wouldn't be used -  these three the tags are in the dictionary, e.g. https://github.com/dcmjs-org/dcmjs/blob/master/src/dictionary.js#L30767. It's not a big deal, but it would be nice to have things localized.

If you need the hex value it would be nice to add a method to the DicomMetaDictionary for that (like the [punctuate and unpunctuate](https://github.com/dcmjs-org/dcmjs/blob/master/src/DicomMetaDictionary.js#L14-L28) ones).

Reply:
I added methods and tests to the DicomMetaDictionary that we can use it to get the tags by name. Now, we get the tags from the dictionary.js. I hope that was, what you had in mind.

Reply:
Yes, thanks!